.github/workflows/e2e-tests-build-docker-image.yml

```diff
@@ -0,0 +1,16 @@
+name: Docker Image Builder For E2E Tests
+
+on:
+  workflow_dispatch:
+
+jobs:
+  docker-build-e2e:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build E2E Docker Image
+        uses: ./.github/actions/docker-build-e2e
+        with:
+          name: oisy-e2e-image
```

Comment:
isn't the upload of the image missing here? or am i missing something? honest question

Reply:
This is where an artifact (an archive with e2e docker image) is uploaded - https://github.com/dfinity/oisy-wallet/blob/ci(e2e)/separate-e2e-workflows/.github/actions/docker-build-e2e/action.yml#L44

Reply:
ah! thank you for the explanation! I had missed that part